Sources/IRGen/Expression/IRFunctionCall.swift

```diff
@@ -31,11 +31,13 @@ struct IRFunctionCall {
         .rendered(functionContext: functionContext)
     }
 
-    let args: String = functionCall.arguments.map({ argument in
-      return IRExpression(expression: argument.expression, asLValue: false).rendered(functionContext: functionContext)
-    }).joined(separator: ", ")
+    let (preamble, args) : (String, [String]) = functionCall.arguments.reduce(("", []), { acc, argument in
```

Comment:
You've done this three times now, yet the way you structure it always changes slightly. Now you have an explicit type annotation?